File: CS/SkinsDemo/Site1.Master.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using DevExpress.Web.ASPxEditors;

namespace SkinsDemo
{
    public partial class Site1 : System.Web.UI.MasterPage
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            DataBind();
        }
        /* Skins */
        protected void cbSkins_DataBound(object sender, EventArgs e)
        {
            if (!IsPostBack)
            {
                ListEditItem item = cbSkins.Items.FindByValue(Page.Theme);
                if (item == null)
                    item = cbSkins.Items.FindByValue(BasePage.DefaultThemeName);
                if (item != null)
                    cbSkins.SelectedItem = item;
            }
        }
    }
}

File: VB/SkinsDemo/Site1.Master.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls
Imports DevExpress.Web.ASPxEditors

Namespace SkinsDemo
	Partial Public Class Site1
		Inherits System.Web.UI.MasterPage
		Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
			DataBind()
		End Sub
		' Skins 
		Protected Sub cbSkins_DataBound(ByVal sender As Object, ByVal e As EventArgs)
			If (Not IsPostBack) Then
				Dim item As ListEditItem = cbSkins.Items.FindByValue(Page.Theme)
				If item Is Nothing Then
					item = cbSkins.Items.FindByValue(BasePage.DefaultThemeName)
				End If
				If item IsNot Nothing Then
					cbSkins.SelectedItem = item
				End If
			End If
		End Sub
	End Class
End Namespace
